docs/src/main/sphinx/connector/row-decoding.fragment

```diff
@@ -0,0 +1,322 @@
+Row decoding
+^^^^^^^^^^^^
+
+For key and message, a decoder is used to map message and key data onto table columns.
+
+The Kinesis connector contains the following decoders:
```

Comment:
Need to remove "Kinesis" from this fragment if it's to be reused in Redis and Kafka as well. Suggest a generic "The connector" like we've used in other fragments, and in the list below just "Message is..."

Let's see how that looks. We may also want to break this up into more, smaller fragments specific to each decoder and leave the parent section intact for connector-specific content.